chromium/base/process/process_metrics_mac.cc assumes PAGE_SIZE is a constexpr

diff --git a/src/3rdparty/chromium/base/process/process_metrics_mac.cc b/src/3rdparty/chromium/base/process/process_metrics_mac.cc
index fb91414..4400ece 100644
--- a/src/3rdparty/chromium/base/process/process_metrics_mac.cc
+++ b/src/3rdparty/chromium/base/process/process_metrics_mac.cc
@@ -296,12 +296,8 @@ bool GetSystemMemoryInfo(SystemMemoryInfoKB* meminfo) {
   }
   DCHECK_EQ(HOST_VM_INFO64_COUNT, count);
 
-#if defined(ARCH_CPU_ARM64)
-  // PAGE_SIZE is vm_page_size on arm, which isn't constexpr.
   DCHECK_EQ(PAGE_SIZE % 1024, 0u) << "Invalid page size";
-#else
-  static_assert(PAGE_SIZE % 1024 == 0, "Invalid page size");
-#endif
+
   meminfo->free = saturated_cast<int>(
       PAGE_SIZE / 1024 * (vm_info.free_count - vm_info.speculative_count));
   meminfo->speculative =
